subsys/bluetooth/mesh/adv_legacy.c

```diff
@@ -95,7 +95,9 @@ static inline void adv_send(struct net_buf *buf)
 	param.interval_min = BT_MESH_ADV_SCAN_UNIT(adv_int);
 	param.interval_max = param.interval_min;
 
+#if BT_DBG_ENABLED
 	uint64_t time = k_uptime_get();
+#endif
```

Comment:
@jhedberg I originally wanted to solve the warning problem, but it is not feasible now. I want to add `(void) time`. Is this OK?